analyzer/javacg-opal/src/test/java/eu/fasten/analyzer/javacgopal/CGandStitchingTest.java

```diff
@@ -102,6 +103,12 @@ public void staticInitializer() throws OPALException, URISyntaxException {
             "\n", toString(mergedRcg));
     }
 
+    @Test
+    public void noPackage() throws URISyntaxException {
+        final var np = getRCG("merge/staticInitializer/NoPackage.class", "noPackage", "0.0.0");
```

Comment:
Shouldn't this be "NoPackage"? (capital case N)

Reply:
fixed